subsys/bluetooth/mesh/scheduler_srv.c

```diff
@@ -729,6 +778,10 @@ int bt_mesh_scheduler_srv_time_update(struct bt_mesh_scheduler_srv *srv)
 		return -EINVAL;
 	}
 
+	for (uint8_t idx = 0; idx < BT_MESH_SCHEDULER_ACTION_ENTRY_COUNT; ++idx) {
```

Comment:
Use `int` for iterators. Conventionally, stdint types should only be used in for loops like this if the bitwidth is of importance. This allows us to communicate intent for these special scenarios. Using `uint8_t` iterators also forces the compiler to add an extra masking instruction after the increment step of the loop, as the processor only operates on 32 bit integers.